contracts/stellar-interchain-token-service/INTEGRATION.md

```diff
@@ -0,0 +1,41 @@
+# Interchain Token Service Integration
+
+## ITS Hub
+
+The Stellar implementation of the Interchain Token Service (ITS) communicates with other ITS contracts through the [ITS Hub](https://docs.axelar.dev/dev/amplifier/its-hub/introduction/), a Cosmwasm smart contract deployed on the Axelar network which acts as central routing hub for interchain token transfers. All interchain token functionality, such as transfering tokens or deploying Interchain Token contracts is achieved via messages sent to or received from the ITS Hub. Direct communication between Stellar ITS and ITS contracts on other chains is not supported.
+
+## Differences between Stellar ITS and EVM ITS
+
+### Contract size restrictions
+
+Stellar allows significantly larger contract code size than EVM. This allows some functionality to be contained within the Stellar ITS contract without the need for splitting into smaller contracts:
+
+- Stellar ITS has no Factory contract. Functionality such as deploying token contracts, registering canonical tokens, and computing token IDs and deployment salts is handled directly by the ITS contract.
+- Flow limit is tracked and enforced by the ITS contract directly rather than by individual Token Managers.
+
+### Authorization
+
+Soroban's authorization model differs slightly from EVM, as there is no notion of a `msg.sender`. If a function requires authorizing the caller, the calling address must be passed as a parameter.
+
```

Comment:
discuss the interchain token contract that's deployed by ITS, since it's not a classic asset

Reply:
discuss how the recipient and minter needs to be encoded when sending a msg from another ITS edge